andela/bp-esa-backend
# #160436728 Create channels for new engagements

#### What does this PR do?
Enables the creation of new slack channels for a new partner engagement.

#### Description of Task to be completed?
When a developer is placed, `p-partner` and `p-partner-int` slack channels should be created. This happens in the event it's a new partner.

#### How should this be manually tested?

Assuming you are an administrator of the workspace:
- Create an [app](https://api.slack.com/apps). It should have workspace privileges in order to create channels(`groups:write`) 
- Generate a token for the app and add it to the `.env` file as the `SLACK_TOKEN`
- Calling the `createPartnerChannels` function in `server/modules/slackIntegration.js`, while providing the partner's details, will create a **private** channel for a specific partner engagement

#### What are the relevant pivotal tracker stories?
[160436728](https://www.pivotaltracker.com/story/show/160436728)
